internal/clients/connect/content.go

```diff
@@ -83,3 +102,48 @@ func ConnectContentFromConfig(cfg *config.Config) *ConnectContent {
 	}
 	return c
 }
+
+func ConnectContentSummaryFromServer(account *accounts.Account, log logging.Logger) ([]*ConnectContentSummary, error) {
+	client, err := NewConnectClient(account, 30*time.Second, events.NewNullEmitter(), log)
+	if err != nil {
+		return nil, err
+	}
+	contentItems, err := client.GetContent(log)
+	if err != nil {
+		return nil, err
+	}
+	count := 0
+	for _, content := range *contentItems {
+		if content.Role == "owner" || content.Role == "editor" {
+			count += 1
+		}
+	}
+	summary := make([]*ConnectContentSummary, count)
+	nextIndex := 0
+	for _, content := range *contentItems {
+		if content.Role == "owner" || content.Role == "editor" {
+			s := &ConnectContentSummary{
+				GUID:            content.GUID,
+				Name:            content.Name,
+				Created:         content.Created,
+				LastDeployed:    content.LastDeployed,
+				AppMode:         content.AppMode,
+				ContentCategory: content.ContentCategory,
+				DashboardURL:    content.DashboardURL,
+				Role:            content.Role,
+				OwnerGUID:       content.OwnerGUID,
+			}
+			title, set := content.Title.Get()
+			if set {
+				s.Title = title
+			}
+			valid := content.BundleId.Valid()
+			if valid {
```

Comment:
Since both `BundleId` fields are `Nullint64Str` type, you can assign it as part of the object being created above without checking the `Valid` flag.